README.md

```diff
@@ -1,46 +1,67 @@
-> FLEDGE has been renamed to Protected Audience API. To learn more about the name change, see the [blog post](https://privacysandbox.com/intl/en_us/news/protected-audience-api-our-new-name-for-fledge)
-
-# Protected Audience Services documentation
+# Protected Auction Services documentation
 
 This GitHub repository contains proposals for the Privacy Sandbox's
-Protected Audience server-side infrastructure that supports the Privacy
+Protected Auction server-side infrastructure that supports the Privacy
 Sandbox's client-side APIs.
 
+Protected Auction includes [Protected Audience](https://github.com/WICG/turtledove/blob/main/FLEDGE.md)
+and [Protected App Signals](https://developer.android.com/design-for-safety/privacy-sandbox/protected-app-signals)
+ad targeting products.
+
 > [!IMPORTANT]
 > This repository is for documentation only. For discussions, visit the [WICG/protected-auction-services-discussion](https://github.com/WICG/protected-auction-services-discussion) repository.
 
-## Protected Audience services overview
+## Protected Auction services trust model
 
-* [Overview of Privacy Sandbox Protected Audience services](trusted_services_overview.md)
+* [Overview of trust model](trusted_services_overview.md)
 
 ## Bidding and Auction services
 
 * [Bidding and Auction services](bidding_auction_services_api.md)
+* [Bidding and Auction services: Protected App Signals Ad Selection](bidding_auction_services_protected_app_signals.md)
 * [Bidding and Auction services onboarding and self-serve guide](bidding_auction_services_onboarding_self_serve_guide.md)
 * [Bidding and Auction services payload optimization](https://github.com/privacysandbox/fledge-docs/blob/main/bidding-auction-services-payload-optimization.md)
 * [Bidding and Auction services system design](bidding_auction_services_system_design.md)
 * [Bidding and Auction services multi seller auctions](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_multi_seller_auctions.md)
-* [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
-* [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 * [Protected Audience event level reporting with Bidding and Auction services](bidding_auction_event_level_reporting.md)
+* Cloud support
+    * AWS: [Bidding and Auction services AWS cloud support and deployment guide](bidding_auction_services_aws_guide.md)
+    * GCP: [Bidding and Auction services GCP cloud support and deployment guide](bidding_auction_services_gcp_guide.md)
 
 ## Key/Value service
 
 * [Key/Value service trust model](key_value_service_trust_model.md)
-* [Key/Value service user-defined functions (UDFs)](key_value_user_defined_functions.md)
+* [Key/Value service use cases](key_value_service_use_cases.md)
+* [Key/Value service user-defined functions (UDFs)](key_value_service_user_defined_functions.md)
 * [Key/Value sharding](key_value_sharding.md)
```

Comment:
This file seems to have a different name now.

Reply:
Yeah, the KV sharding link changed. Updated here.